### PR TITLE
feat: Adds support for image resize

### PR DESCRIPTION
**Pull request by iamsivin**

### CODE REVIEW

File: src/schema/markdown/serializer.js [hunk 1]
```diff
@@ -48,11 +48,16 @@ export const paragraph = (state, node) => {
   state.closeBlock(node);
 };
 export const image = (state, node) => {
+  let src = state.esc(node.attrs.src);
+  const hasQueryAlready = src && src.indexOf('?') > -1;
+  if (node.attrs.height && !hasQueryAlready) {
+    src += `?cw_image_height=${node.attrs.height}px`;
+  }
```
Comment (scmmishra) [inline src/schema/markdown/serializer.js]: Is it possible to use the URL methods here?

File: src/schema/message.js [hunk 2]
```diff
@@ -11,7 +11,32 @@ export const messageSchema = new Schema({
...
+        ...schema.spec.nodes.get('image').attrs,
+        height: {default: null}
+      },
+      parseDOM: [{
+        tag: 'img[src]',
+        getAttrs: dom => ({
+          src: dom.getAttribute('src'),
+          title: dom.getAttribute('title'),
+          alt: dom.getAttribute('alt'),
+          height: parseInt(dom.style.height, 10)
```
Comment (scmmishra) [inline src/schema/message.js]:
```suggestion
          height: parseInt(dom.style.height)
```

The default is base-10 anyway, no need for this

Comment (scmmishra) [inline src/schema/message.js:26]: Do we need to set this explicitly? What happens if I set it to `null` instead?

File: src/schema/message.js [hunk 3]
```diff
@@ -11,7 +11,32 @@ export const messageSchema = new Schema({
...
+        })
+      }],
+      toDOM: node => {
+        const attrs = {
+          src: node.attrs.src,
+          alt: node.attrs.alt
```
Comment (scmmishra) [inline src/schema/message.js]: Let the height attribute stay here anyway, its helps the browser to determine the layout

File: src/schema/message.js (hunk 2 above)
Comment (iamsivin) [reply src/schema/message.js]: @scmmishra Showing some eslint warning. That's the reason I added it.
image

File: src/schema/markdown/serializer.js (hunk 1 above)
Comment (iamsivin) [reply src/schema/markdown/serializer.js]: @scmmishra Done https://github.com/chatwoot/prosemirror-schema/pull/21/files#diff-df55ca64497de9df74c663f424fc6b01cfcaeb0acb875db40ce8e5a60b047510R53

File: src/schema/message.js (hunk 2 above)
Comment (iamsivin) [reply src/schema/message.js]: Done

File: src/schema/message.js (hunk 3 above)
Comment (iamsivin) [reply src/schema/message.js]: Done

**Review by scmmishra — APPROVED** [review]

**Comment by scmmishra** [comment]
LGTM! Maybe @nithindavid can also take a look

**Pull request merged**